src/main/java/nl/ordina/jobcrawler/service/MatchSkillsService.java

```diff
@@ -0,0 +1,82 @@
+package nl.ordina.jobcrawler.service;
+
+
+import lombok.extern.slf4j.Slf4j;
+import nl.ordina.jobcrawler.model.Skill;
+import nl.ordina.jobcrawler.model.Vacancy;
+import nl.ordina.jobcrawler.repo.SkillRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/*
+*  This service contains the functionality to match skills in the skill table with
+*  vacancies in the vacancy table and to add or update relationship records accordingly:
+*  It relies on the SkillService to interact with the database.
+*
+* This service has the following functionality:
```

Comment:
Move these to the JavaDoc of the specified method or function

Reply:
adapted